src/coreclr/nativeaot/Runtime/arm64/ExceptionHandling.S

```diff
@@ -338,6 +338,93 @@ LOCAL_LABEL(NotHijacked):
         EMIT_BREAKPOINT
     NESTED_END RhpThrowEx, _TEXT
 
+//
+// RhpThrowExact
+//
+// SUMMARY:  Similar to RhpThrowEx, except that it sets the rethrow flag
```

Comment:
We can avoid duplicating the entire code for a different flag: https://github.com/dotnet/runtime/commit/86e5b2128f7f9373c070ebb569b54ed0b8f76f08.